quickwit/quickwit-serve/src/template_api/rest_handler.rs

```diff
@@ -55,6 +56,7 @@ pub(crate) fn index_template_api_handlers(
         .or(update_index_template_handler(metastore.clone()))
         .or(delete_index_template_handler(metastore.clone()))
         .or(list_index_templates_handler(metastore.clone()))
+        .recover(recover_fn)
 }
 
 fn create_index_template_handler(
```

Comment:
this and `list_index_templates_handler` might be missing a `warp::path::end` ?

Reply:
It is implicit in the warp::path! macro.

> The path! macro automatically assumes the path should include an end() filter. To build up a path filter prefix, such that the end() isn’t included, use the / .. syntax.